### ISSUE TITLE
iOS7 and UIView tintColor

### ISSUE THREAD

**Issue by grantjk** [tombenner/nui#223]
Now that in iOS7 all UIView's support a tintColor property, it makes sense to have this option in Nui on UIView.

Before I go trying to write a pull request, is there any reason it's not in here yet?

**Comment by PabloLerma** [comment]
Yeah, we should do it. I see that you haven't finally done it. I will try to do it and make a PR.

**Comment by timbodeit** [comment]
See #298. 90% done, but still needs some work.
Would be awesome if someone could build on top of this.
